dockerfiles/dashboard-dev/Dockerfile

```diff
@@ -8,14 +8,14 @@
 # Contributors:
 #   Red Hat, Inc. - initial API and implementation
 
-FROM node:8.15.0
+FROM node:8.15.1
```

Comment:
maybe we should use 8.16.0 which has a newest yarn version ?

Reply:
Done. Tested with dashboard and workspace loader. Workspace loader uses this image in the dev-file too.